documentation/pom.xml

```diff
@@ -16,32 +16,33 @@
     <description>The Infinispan Documentation</description>
 
     <properties>
-        <infinispan.version>8.0</infinispan.version>
-        
+        <infinispan.version>9.0</infinispan.version>
         <maven.deploy.skip>true</maven.deploy.skip>
-		<skipTests>true</skipTests>
+		    <skipTests>true</skipTests>
     </properties>
 
     <build>
         <plugins>
             <plugin>
                 <groupId>org.asciidoctor</groupId>
                 <artifactId>asciidoctor-maven-plugin</artifactId>
-                <version>0.1.3.1</version> 
+                <version>1.5.3</version> 
                 <executions>
                     <execution>
-                        <id>create-html</id>
-                        <phase>process-resources</phase>
+                        <id>asciidoc</id>
+                        <phase>generate-resources</phase>
                         <goals>
                             <goal>process-asciidoc</goal>
                         </goals>
                     </execution>
                 </executions>
                 <configuration>
-                    <toc>true</toc>
+                    <!--backend>docbook</backend-->
                     <backend>html5</backend>
                     <doctype>book</doctype>
-                    <goalPrefix>asciidoctor</goalPrefix>
+                    <preserveDirectories>true</preserveDirectories>
+                    <relativeBaseDir>true</relativeBaseDir>
+                    <sourceDocumentExtensions>adoc</sourceDocumentExtensions>
```

Comment:
I'm getting an error:

    [ERROR] Failed to execute goal org.asciidoctor:asciidoctor-maven-plugin:1.5.3:process-asciidoc (asciidoc) on project infinispan-docs: Unable to parse configuration of mojo org.asciidoctor:asciidoctor-maven-plugin:1.5.3:process-asciidoc for parameter sourceDocumentExtensions: Cannot assign configuration entry 'sourceDocumentExtensions' with value 'adoc' of type java.lang.String to property of type java.util.List -> [Help 1]

And this fixes it:

                    <sourceDocumentExtensions>
                        <sourceDocumentExtension>adoc</sourceDocumentExtension>
                    </sourceDocumentExtensions>